hw_19042018/customer.rb

```diff
@@ -0,0 +1,27 @@
+load 'person.rb'
+class Customer < Person
+  attr_accessor :indexID , :joindate 
+  def input
+    puts "enter name: "
+    @name = gets.chomp()
+    puts "enter gender: "
+    @gender = gets.chomp()
```

Comment:
Invoke super method to assign `@name` and `@gender`.
DRY(Don't repeat yourself). https://en.wikipedia.org/wiki/Don%27t_repeat_yourself